src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,498 @@
+---
+title: Monitor your Flutter app
+tags:
+    - Flutter
+    - Mobile monitoring
+    - New Relic Mobile Flutter
+    - Get started
+metaDescription: "New Relic's Flutter agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.png'
+
+Monitor your Flutter application with our New Relic Flutter agent. Capture Dart errors, track specific network requests, and use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+
+<img
```

Comment:
This image is REALLY big. I'd make it smaller (like the actual image itself by resizing the file) or putting it into a side-by-side w/ the intro